main.go

```diff
@@ -7,16 +7,20 @@ import (
 	"os"
 	"time"
 
+	crypto "github.com/libp2p/go-libp2p-core/crypto"
+
 	pg "github.com/go-pg/pg/v10"
 	orm "github.com/go-pg/pg/v10/orm"
+
 	ipfsgeoip "github.com/hsanjuan/go-ipfs-geoip"
 	ipfslite "github.com/hsanjuan/ipfs-lite"
-	crypto "github.com/libp2p/go-libp2p-core/crypto"
+
 	multiaddr "github.com/multiformats/go-multiaddr"
 	madns "github.com/multiformats/go-multiaddr-dns"
 )
 
 const schema = "locations"
+const crawlTime = time.Duration(10) * time.Second
```

Comment:
nitpick. I am not sure if this is in any go guidelines or just personal preference.

```suggestion
const crawlTime time.Duration = 10 * time.Second
```